cdap-gateway/src/main/java/co/cask/cdap/gateway/router/handlers/WrappedDiscoverable.java

```diff
@@ -40,6 +40,10 @@ public InetSocketAddress getSocketAddress() {
     return discoverable.getSocketAddress();
   }
 
+  public byte[] getPayload() {
```

Comment:
We should get rid of this class. With twill-0.8.0, this class is no longer necessary (besides the hashCode and equals need to include the payload).